### PR TITLE
Add stats for JUMP cpg0016

### PR DESCRIPTION
**Pull request by AnneCarpenter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ E.g. the accession number of the first dataset is `cpg0000`.
...
 | cpg0012-wawer-bioactivecompoundprofiling | 30,000 compound dataset in U2OS cells                                                                                           | 1,2                 | idr0016                  |
 | cpg0015-heterogeneity                    | 2,200+ compounds and 200+ genes profiles in U2OS cells                                                                          | 8                   | idr0016,idr0036, idr0033 |
-| cpg0016-jump                             | 116,000+ compounds and 16+ genes (CRISPR knockout and overexpression) profiled in U2OS cells                                    | 9                   |                          |
+| cpg0016-jump                             | 116,000+ compounds and 16+ genes (CRISPR knockout and overexpression) profiled in U2OS cells. Over 8 million images (>126 TB), over 1.5 billion cells of numerical data (>126TB), for over 250 TB data in total.
+                                    | 9                   |                          |
```
Comment (ErinWeisbart) [inline README.md:33]: This messes up the formatting of the table. Delete the extra line break so that `| 9` follows right after `total.`

Comment (shntnu) [reply README.md]: Now fixed

**Review by shntnu — APPROVED** [review]
We don't have a good estimate of the size of non-image data, but >126TB is good for now

**Comment by shntnu** [comment]
I'll go ahead and merge @ErinWeisbart (thanks for checking)

**Pull request merged**